### ISSUE TITLE
Changes to the nestjs schematic moving the endpoint to /api can be confusing

### ISSUE THREAD

**Issue by DouglasWebster** [nrwl/nx#1132]
The change to the file packages/schematics/src/collection/node-application/files/nestjs/main.ts__tmpl__ can lead to a little confusion as the message printed will be

Listening at http://localhost:3333

or whatever the port has been set to.

Now that you are setting a global prefix would it not be better to print out.

Listening at http://localhost:3333/api

On a new project, the first time I ran the nest server I went to the indicated end point and saw the dreaded 404 message and I then had to inspect the code to find out where the problem lay.
I amended my code so that the message points to the correct end point. The amended code is bellow; not suggesting that this is the code that should be adopted but something similar may prevent confusion in the future.
```
async function bootstrap() {
  const app = await NestFactory.create(AppModule);
  const globalPrefix = "api";
  app.setGlobalPrefix(globalPrefix);
  const port = process.env.port || 3333;
  await app.listen(port, () => {
     console.log(`Listening at http://localhost:${port}/${globalPrefix}`);
  });
}
```

**Comment by FrozenPandaz** [comment]
I agree 💯. This should be a good first PR. Does someone want to give it a shot?

**Comment by FrozenPandaz** [comment]
Fixed with #1201
